packages/app/src/app/pages/common/UserMenu/index.js

```diff
@@ -1,14 +1,16 @@
 import * as React from 'react';
-import { inject, observer } from 'mobx-react';
+import { observer } from 'mobx-react-lite';
 
 import HoverMenu from 'app/components/HoverMenu';
 import Relative from '@codesandbox/common/lib/components/Relative';
+import { useSignals, useStore } from 'app/store';
 
 import Menu from './Menu';
 import { ClickableContainer, ProfileImage } from './elements';
 
-function UserMenu({ signals, store }) {
-  const { user, userMenuOpen } = store;
+function UserMenu() {
+  const { user, userMenuOpen } = useStore();
+  const signals = useSignals();
```

Comment:
@SaraVieira I think you should destructur `signals`,  just to be consistent with the rest of the codebase

```suggestion
  const { files: { gotUploadedFiles }, modalOpened, userMenuClosed, signOutClicked, userMenuOpened } = useSignals();
```

Reply:
Done! And also made some functions implict returns